contents/navigation/paginations/centered-full-width/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "en": {
+    "title": "Centered full width",
+    "description": "Centered full width"
+  },
+  "ja": {
+    "title": "横幅最大中央寄せ",
+    "description": "横幅最大中央寄せ"
+  },
+  "options": {
+    "container": {
+      "p": "md"
+    }
```

Comment:
How about adding this and placing it in the center?
```json
"centerContent": true,
```
![image](https://github.com/yamada-ui/yamada-components/assets/60034520/c2bb00a3-71d0-488d-8250-de8acbbcd7b4)

Please correct the same in other areas.

Reply:
@illionillion 

thx!
I fixed it.
9bf5b6109bd59a7980034700926623f1ae6f8fbf

I've changed the style a bit.